OracleDataIntegrator/README.md

```diff
@@ -121,7 +121,7 @@ Now you can access the Agent at http://<host name>:20910/oraclediagent
          
 **NOTES:** 
 
-1) If DB_SCHEMA_PASSWORD, SUPERVISOR_PASSWORD, WORK_REPO_PASSWORD are not provided in odi.env.list then it will generate random password and use it while running RCU. It will display generated random password on console
+1) If DB_SCHEMA_PASSWORD, SUPERVISOR_PASSWORD, WORK_REPO_PASSWORD are not provided in odi.env.list then it will generate random password and use it while running RCU. It will display generated random password on console or can be found by looking in the Docker logs
```

Comment:
Please provide the command required to output the logs and how to find the password in those logs.